shell/browser/electron_browser_client.cc

```diff
@@ -649,6 +720,23 @@ void ElectronBrowserClient::AppendExtraCommandLineSwitches(
   std::string process_type =
       command_line->GetSwitchValueASCII(::switches::kProcessType);
 
+#if defined(OS_LINUX)
+  bool enable_crash_reporter = false;
+  enable_crash_reporter = breakpad::IsCrashReporterEnabled();
+  if (enable_crash_reporter) {
+    command_line->AppendSwitch(::switches::kEnableCrashReporter);
+    std::string switch_value;
+    for (const auto& pair : api::crash_reporter::GetGlobalCrashKeys()) {
+      if (!switch_value.empty())
+        switch_value += ",";
+      switch_value += pair.first;
+      switch_value += "=";
+      switch_value += pair.second;
```

Comment:
Is there a size limit to the values, which should be warned to users ?

Reply:
yes, but this isn't where the size limit appears. Crash key values are set to max out at 128 bytes [here](https://github.com/electron/electron/pull/23265/files#diff-479c9f061dab9becb139549a1177ff45R26). I believe key names will max out at 64 bytes in crashpad, not sure about breakpad & haven't tested either.

We should definitely communicate these limits in the docs.